.tool-versions

```diff
@@ -1,2 +1,2 @@
-ruby 2.6.5
-nodejs 10.15.3
+ruby 2.6.6
+nodejs 12.18.2
```

Comment:
There are newer versions of Ruby and Node, but this is where my blog is at, and I know these work. Maybe once I update my blog, I'll come back and update them. 🙂